pkg/agent/openflow/cookie/allocator.go

```diff
@@ -104,6 +106,9 @@ func (i ID) String() string {
 type Allocator interface {
 	// Request cookie IDs of flow categories.
 	Request(cat Category) ID
+	// Request cookie IDs of flow categories and populates mask to reserved bits.
+	// Only last 48bits (BigEndian) of the mask will be used.
+	RequestMask(cat Category, mask uint64) ID
```

Comment:
I think it's 40 bits? but why don't allocate bits for purposes like it did before? I think allocating 40bits helps neither 32bits data nor 64 bits data. It wastes 8 bits for the former and cut 24 bits for the latter.
Since you only need 32 bits currently, how about just use the last 32 bits as the object ID and still keep 8bits for reservation?
```
RequestMask(cat Category, objectID uint32)
```

Reply:
My fault, it is 40bits.
Updated.